optaplanner-benchmark/src/test/resources/org/optaplanner/benchmark/api/classloaderTestdataConstraints.drl

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.optaplanner.examples.cloudbalancing.solver;
+package org.optaplanner.benchmarker.api;
```

Comment:
Thanks for fixing the package.

For the location of this file (moving it from core to benchmark): if we're doing a solver+drl integration test in optaplanner-solver and a benchmark+drl integration test in optaplanner-benchmark, they could still reuse the same drl file by having a "test-jar" dependency from optaplanner-benchmark on optaplanner-solver.